README.md

```diff
@@ -10,6 +10,38 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
 
 > Neural Speed is under active development so APIs are subject to change.
 
+## Quick Start
+There are two methods for utilizing the Neural Speed:
+You can refer to [ITREX(intel extension for transformers)](https://github.com/intel/intel-extension-for-transformers) for [Transformer-based API](https://github.com/intel/intel-extension-for-transformers/blob/main/docs/weightonlyquant.md#llm-runtime-example-code). 
+
+Or you can use Neural Speed straight forward:
+
+#### One-click Python scripts
+Run LLM with one-click python script including conversion, quantization and inference.
+```
+python scripts/run.py model-path --weight_dtype int4 -p "She opened the door and see"
+```
+
+#### Quantize and Inference Step By Step
+Besides the one-click script, Neural Speed also offers the detailed script: 1) convert and quantize, and 2) inference.
+
+##### 1. Convert and Quantize LLM
+Neural Speed assumes the compatible model format as [llama.cpp](https://github.com/ggerganov/llama.cpp) and [ggml](https://github.com/ggerganov/ggml). You can also convert the model by following the below steps:
+
+```bash
+# convert the model directly use model id in Hugging Face. (recommended)
+python scripts/convert.py --outtype f32 --outfile ne-f32.bin EleutherAI/gpt-j-6b
+```
+
+##### 2. Inference
+
+We provide LLM inference script to run the quantized model. Please reach [us](mailto:itrex.maintainers@intel.com) if you want to run using C++ API directly.
```

Comment:
do we have neural speed maintainer  email ？

Reply:
we don't have it now